src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
@leleuj so should I wait for those changes to land in 1.9.6-SNAPSHOT or will you do that only in pac4J 2.x?